### PR TITLE
Common UI - Top Scroll Navigation

### PR DESCRIPTION
**Pull request by Umasankar1522**

### CODE REVIEW

File: src/components/TopScrollNavigation/TopScroll.component.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React, {useEffect, useState } from 'react';
+
+const arrowUpImage = 'https://www.jaypore.com/images/arrow-up-circle.png'; // Arrow mark image
```
Comment (prateekoli9) [inline src/components/TopScrollNavigation/TopScroll.component.tsx:3]: Save image as an asset (in assets/images folder) and use from there

File: src/components/TopScrollNavigation/TopScroll.component.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const pageOffsetValue = 500; // pageYOffsetValue to display the Back to Top arrow mark
```
Comment (prateekoli9) [inline src/components/TopScrollNavigation/TopScroll.component.tsx:4]: in helpers folder create new file constants.ts, export this constant from there

File: src/components/TopScrollNavigation/TopScroll.component.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            setShowTopBtn(false);
+        }
+    };
+
+    useEffect(() => {
+        window.addEventListener('scroll', checkScrollTop);
+        return () => window.removeEventListener('scroll', checkScrollTop);
+    }, [checkScrollTop]);
+   
+    const backToTopStyles: React.CSSProperties = {
```
Comment (prateekoli9) [inline src/components/TopScrollNavigation/TopScroll.component.tsx:28]: move styles to TopScroll.module.scss file

Comment (Umasankar1522) [reply src/components/TopScrollNavigation/TopScroll.component.tsx:28]: moved the TopScroll component styles to TopScroll.module.scss

File: src/components/TopScrollNavigation/TopScroll.component.tsx (hunk 1 above)
Comment (Umasankar1522) [reply src/components/TopScrollNavigation/TopScroll.component.tsx:3]: Arrow_up image is saved in Assets folder and importing it from there to component

**Pull request merged**